Pyrado/scripts/training/qq-su_udr-grav_ppo.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2020, Fabio Muratore, Honda Research Institute Europe GmbH, and
```

Comment:
Again, only one qq-su_udr-ppo.py file. Maybe work with args from the command line to run different flavors.